app/Models/User.php

```diff
@@ -47,7 +50,35 @@ class User extends Authenticatable
         'password' => 'hashed',
     ];
 
-    public function tasks(): BelongsToMany{
-        return $this -> belongsToMany(Task::class);
+    public function tasks(): BelongsToMany
+    {
+        return $this->belongsToMany(Task::class);
+    }
+
+    /**
+     * Get the user's full name.
+     *
+     * @return string
+     */
+    public function getFullNameAttribute()
+    {
+        return "{$this->first_name} {$this->last_name}";
```

Comment:
Này nhìn hơi kì nhỉ ?
Để thế này cũng đc hoặc sửa theo suggestion của anhes
```suggestion
        return $this->first_name . ' ' . $this->last_name
```